react-frontend/src/components/Guides/CollaborationTools/collaborationTools.js

```diff
@@ -1,17 +1,55 @@
 import React from 'react';
-
+import styled from 'styled-components';
+import { createGlobalStyle } from 'styled-components';
 import CollaborationTool from './collaborationTool';
-// import ResourceHeader from './resourceheader';
-// import '@bcgov/bc-sans/css/BCSans.css'
-import '../../../css/collaborationTools.css';
+
+const GlobalStyle = createGlobalStyle`
+ html {
+  font-size: 10px;
+ }
+`;
+const CollabToolContainer = styled.div`
+  padding-top: 120px;
+  font-size: 2rem;
+  max-width: 80rem;
+  margin-right: auto;
+  margin-left: auto;
+  padding-left: 15px;
+  padding-right: 15px;
+`;
+const Title = styled.h1`
+  font-size: 3rem;
+  font-weight: 700;
+  margin-top: 3rem;
+  width: 100%;
+`;
+const Title2 = styled.h2`
+  font-size: 3rem;
+  font-weight: 700;
+  margin-top: 3rem;
+  width: 100%;
+`;
+const Title3 = styled.h3`
+  font-size: 2.5rem;
+  font-weight: 700;
+  margin-top: 3rem;
+  width: 100%;
```

Comment:
all heading tags are already `block` elements and so are always `100%` wide

Reply:
hmm weird that that was redundantly declared on the wp site.